src/frontends/tensorflow/src/frontend.cpp

```diff
@@ -195,15 +203,33 @@ ov::frontend::InputModel::Ptr FrontEnd::load_impl(const std::vector<ov::Any>& va
 std::shared_ptr<ov::Model> FrontEnd::convert(const ov::frontend::InputModel::Ptr& model) const {
     auto f = convert_partially(model);
 
-    auto unsupported_operations = get_unconverted_types_from_model(f);
-    if (m_telemetry) {
-        for (const auto& unsupported_operation : unsupported_operations) {
+    std::unordered_map<std::string, std::string> failures;
+    std::vector<std::string> unsupported_operations;
+    get_unsupported_operations_and_failures(f, unsupported_operations, failures);
+
+    std::string exception_message;
+    for (const auto& failure : failures) {
+        if (m_telemetry) {
+            // TODO: 105173 support anonymization of exception message in order to send to telemetry
+        }
+        exception_message += "[TensorFlow Frontend] Internal error: conversion is failed for " + failure.first +
+                             " operation with a message:\n" + failure.second + "\n";
+    }
+
+    // append unsupported operations
+    for (const auto& unsupported_operation : unsupported_operations) {
+        if (m_telemetry) {
             m_telemetry->send_event("error_cause", "tf_" + unsupported_operation);
         }
+        // TODO 107500: report the full list of unsupported operations
     }
-    FRONT_END_OP_CONVERSION_CHECK(
-        unsupported_operations.size() == 0,
-        "[TensorFlow Frontend] Internal error: No translator found for " + unsupported_operations[0] + " node.");
+    exception_message = (unsupported_operations.size() > 0
+                             ? exception_message + "[TensorFlow Frontend] Internal error: No translator found for " +
+                                   unsupported_operations[0] + " node."
```

Comment:
Even if there are no messages collected, there can be multiple operations unsupported, but this code reports about the first unsupported operation only.

Reply:
Will be addressed in the ticket 107500